nixos/modules/services/misc/home-assistant.nix

```diff
@@ -287,7 +287,7 @@ in {
           # Components that require access to serial devices (/dev/tty*)
           # List generated from home-assistant documentation:
           #   git clone https://github.com/home-assistant/home-assistant.io/
-          #   cd source/_integrations
+          #   cd ${src.name}/_integrations
```

Comment:
Please drop this change.

Reply:
I'm going to try and factor out all of the package updates to separate PRs, eventually this should just be the fetcher change

Reply:
I guess I could mark this as draft until then

Reply:
This is editing a comment that isn't fetcher related at all.

Reply:
correct, but eventually I'll fix it when pruning the commits :)